integration/integration_test.go

```diff
@@ -164,268 +167,333 @@ func TestBiotop(t *testing.T) {
-		deleteTestPod,
+		waitUntilTestPodReadyCommand(ns),
+		deleteTestNamespaceCommand(ns),
 	}
 
 	runCommands(commands, t)
 }
 
 func TestExecsnoop(t *testing.T) {
+	ns := "test-execsnoop"
+
+	t.Parallel()
+
 	execsnoopCmd := &command{
 		name:           "Start execsnoop gadget",
-		cmd:            "$KUBECTL_GADGET execsnoop -n test-ns",
-		expectedRegexp: `test-ns\s+test-pod\s+test-pod\s+date`,
+		cmd:            fmt.Sprintf("$KUBECTL_GADGET execsnoop -n %s", ns),
+		expectedRegexp: `test-execsnoop\s+test-pod\s+test-pod\s+date`,
 		startAndStop:   true,
 	}
 
 	commands := []*command{
+		createTestNamespaceCommand(ns),
 		execsnoopCmd,
 		{
 			name:           "Run pod which does a lot of exec",
-			cmd:            busyboxPodCommand("while true; do date; done"),
+			cmd:            busyboxPodCommand(ns, "while true; do date; done"),
 			expectedString: "pod/test-pod created\n",
 		},
-		waitUntilTestPodReady,
-		deleteTestPod,
+		waitUntilTestPodReadyCommand(ns),
+		deleteTestNamespaceCommand(ns),
 	}
 
 	runCommands(commands, t)
 }
 
 func TestMountsnoop(t *testing.T) {
+	ns := "test-mountsnoop"
+
+	t.Parallel()
+
 	mountsnoopCmd := &command{
 		name:           "Start mountsnoop gadget",
-		cmd:            "$KUBECTL_GADGET mountsnoop -n test-ns",
+		cmd:            fmt.Sprintf("$KUBECTL_GADGET mountsnoop -n %s", ns),
 		expectedRegexp: `test-pod\s+test-pod\s+mount.*mount\("/mnt", "/mnt", .*\) = -ENOENT`,
 		startAndStop:   true,
 	}
 
 	commands := []*command{
+		createTestNamespaceCommand(ns),
 		mountsnoopCmd,
 		{
 			name:           "Run pod which tries to mount a directory",
-			cmd:            busyboxPodCommand("while true; do mount /mnt /mnt; done"),
+			cmd:            busyboxPodCommand(ns, "while true; do mount /mnt /mnt; done"),
 			expectedRegexp: "pod/test-pod created",
 		},
-		waitUntilTestPodReady,
-		deleteTestPod,
+		waitUntilTestPodReadyCommand(ns),
+		deleteTestNamespaceCommand(ns),
 	}
 
 	runCommands(commands, t)
 }
 
 func TestNetworkpolicy(t *testing.T) {
+	ns := "test-networkpolicy"
+
+	t.Parallel()
+
 	commands := []*command{
+		createTestNamespaceCommand(ns),
 		{
 			name:           "Run test pod",
-			cmd:            busyboxPodCommand("while true; do wget -q -O /dev/null https://kinvolk.io; done"),
+			cmd:            busyboxPodCommand(ns, "while true; do wget -q -O /dev/null https://kinvolk.io; done"),
 			expectedRegexp: "pod/test-pod created",
 		},
-		waitUntilTestPodReady,
+		waitUntilTestPodReadyCommand(ns),
 		{
 			name:           "Run network-policy gadget",
-			cmd:            "$KUBECTL_GADGET network-policy monitor -n test-ns --output ./networktrace.log & sleep 15; kill $!; head networktrace.log",
-			expectedRegexp: `"type":"connect".*"test-ns".*"test-pod"`,
+			cmd:            "$KUBECTL_GADGET network-policy monitor -n test-networkpolicy --output ./networktrace.log & sleep 15; kill $!; head networktrace.log",
+			expectedRegexp: `"type":"connect".*"test-networkpolicy".*"test-pod"`,
```

Comment:
The first one I forgot too but the second one was "on purpose" (and the same for all inside `expectedRegexp`).
Indeed, I use raw string for the regex and I was not sure about how the string will render if I use Fmt.Sprintf() on this raw string because I would like to have a raw string in output.